widgetastic_manageiq/__init__.py

```diff
@@ -4051,3 +4051,461 @@ def data_for_timestamp(self, timestamp):
         for row in self.tooltip.rows():
             tooltip_data.update({row[0].text: row[1].text})
         return tooltip_data
+
+# Widget and its form views for Infra Planning Wizard
+
+
+class MultiSelectBox(Widget):
```

Comment:
**REQUIRED**
We already have `MultiBoxSelect`. Please use another name to avoid mistakes.